test-app/config/ember-try.js

```diff
@@ -53,7 +53,10 @@ module.exports = async function () {
         name: 'ember-canary',
         npm: {
           devDependencies: {
-            'ember-source': await getChannelURL('canary'),
+            'ember-source': await getChannelURL('canary').then((url) => {
+              console.log('canary channel url', url);
```

Comment:
Did you intend to leave this console.log in?

Reply:
indeed no, that was mistake and was fixed in https://github.com/ember-modifier/ember-modifier/pull/688/files